compiler-rt/test/fuzzer/ulimit.test

```diff
@@ -1,4 +1,5 @@
 # FIXME: Disabled on Windows for now because Windows has no ulimit command.
+REQUIRES: shell
 UNSUPPORTED: target={{.*windows.*}}
```

Comment:
Remove `windows` and FIXME

Reply:
Maybe the fixme should stay(or be updated), since we could remove REQUIRES: shell, without the use of ulimit? WDYT?